Doc/library/argparse.rst

```diff
@@ -2002,7 +2002,15 @@ Exiting methods
 .. method:: ArgumentParser.exit(status=0, message=None)
 
    This method terminates the program, exiting with the specified *status*
-   and, if given, it prints a *message* before that.
+   and, if given, it prints a *message* before that. The user can override
+   this method to handle these steps differently::
+
+    import sys
+    class ErrorCatchingArgumentParser(argparse.ArgumentParser):
```

Comment:
Could you reformat your code and use the f-string for your print()?

Thank you

Reply:
Hi, stéphane, your mean Reformating my code by `>>>`?
If user use python's version is lower than 3.6, looks like they can not use f-string
REF: https://www.python.org/dev/peps/pep-0498/